systemtests/src/main/java/com/github/streamshub/systemtests/setup/StrimziOperatorSetup.java

```diff
@@ -0,0 +1,306 @@
+    private static final Logger LOGGER = LogWrapper.getLogger(StrimziOperatorSetup.class);
+    private static final KubeResourceManager RESOURCE_MANAGER = KubeResourceManager.get();
+
+    private final List<File> strimziResources = fetchStrimziYamlFiles();
+    private final String deploymentNamespace;
+    private final String deploymentName;
+
+    public StrimziOperatorSetup(String deploymentNamespace) {
+        this.deploymentNamespace = deploymentNamespace;
+        this.deploymentName = Environment.STRIMZI_OPERATOR_NAME;
+    }
+
+    public void setup() {
+        LOGGER.info("----------- Setup Strimzi Cluster Operator -----------");
+        // Watch all namespaces CRB
+        // https://strimzi.io/docs/operators/latest/deploying#deploying-cluster-operator-to-watch-whole-cluster-str
+        RESOURCE_MANAGER.createResourceWithWait(getStrimziWatchAllCrbs());
+
+        if (Environment.SKIP_STRIMZI_INSTALLATION || ResourceUtils.getKubeResource(Deployment.class, this.deploymentNamespace, this.deploymentName) != null) {
+            LOGGER.warn("Skipping Strimzi installation. It is already installed or env to skip installation was set to `true`!");
+            return;
+        }
+
+        LOGGER.info("Install Strimzi Cluster Operator in namespace: {}", this.deploymentNamespace);
+        RESOURCE_MANAGER.createResourceWithoutWait(getBundleCrds());
+        RESOURCE_MANAGER.createResourceWithoutWait(getBundleClusterRoles());
+        RESOURCE_MANAGER.createResourceWithoutWait(getBundleServiceAccount());
+        RESOURCE_MANAGER.createResourceWithoutWait(getBundleRoleBindings());
+        RESOURCE_MANAGER.createResourceWithoutWait(getBundleClusterRoleBindings());
+        RESOURCE_MANAGER.createResourceWithoutWait(getBundleConfigMap());
+        RESOURCE_MANAGER.createResourceWithWait(getBundleDeployment());
+    }
+
+    public void teardown() {
+        LOGGER.info("----------- Teardown Strimzi Cluster Operator -----------");
+        if (Environment.SKIP_STRIMZI_INSTALLATION || ResourceUtils.getKubeResource(Deployment.class, this.deploymentNamespace, this.deploymentName) == null) {
+            LOGGER.warn("Skipping Strimzi deletion. It is already deleted or installation was skipped!");
+            return;
+        }
+
+        // Watch all namespaces CRB
+        RESOURCE_MANAGER.deleteResource(getStrimziOperatorGroup());
+        RESOURCE_MANAGER.deleteResource(getStrimziWatchAllCrbs());
+        RESOURCE_MANAGER.deleteResource(getBundleCrds());
+        RESOURCE_MANAGER.deleteResource(getBundleClusterRoles());
+        RESOURCE_MANAGER.deleteResource(getBundleServiceAccount());
+        RESOURCE_MANAGER.deleteResource(getBundleRoleBindings());
+        RESOURCE_MANAGER.deleteResource(getBundleClusterRoleBindings());
+        RESOURCE_MANAGER.deleteResource(getBundleConfigMap());
+        RESOURCE_MANAGER.deleteResource(getBundleDeployment());
+    }
+
+    private ClusterRoleBinding[] getStrimziWatchAllCrbs() {
```

Comment:
Some of these bindings are in the Strimzi resources aren't they? Maybe there are differences that I'm not spotting though.

Reply:
I think i had to resort to this because of strimzi not watching all namespaces, not by default only using STRIMZI_NAMESPACE:"*" i think. Follows ->
https://strimzi.io/docs/operators/latest/deploying#deploying-cluster-operator-to-watch-whole-cluster-str